pyanaconda/ui/gui/spokes/advstorage/dasd.py

```diff
@@ -84,63 +90,71 @@ def on_start_clicked(self, *args):
         self._startButton.hide()
         self._cancelButton.set_sensitive(False)
         self._okButton.set_sensitive(False)
-
         self._conditionNotebook.set_current_page(1)
+        self._set_configure_sensitive(False)
```

Comment:
Please move `_set_configure_sensitive` above line `_conditionNotebook.set_current_page` to the others `set_sensitive`.